### ISSUE TITLE
translate models from Multilevel Structural Equation Modeling, Online Appendix

### ISSUE THREAD

**Issue by jpritikin** [OpenMx/OpenMx#246]
http://levente.littvay.hu/msem/

**Comment by jpritikin** [comment]
I started working on this. I couldn't get [Fig 2.4](https://github.com/OpenMx/sbl/blob/master/Chapter%202/fig_2.4.R) to match. I believe I translated it correctly, but the -2ll is off by about 300 points and the Mplus parameter values result in a -2ll that is much worse than it should be. Any idea what is going on?

**Comment by jpritikin** [comment]
The parameter estimates match for [Fig 3.3](https://github.com/OpenMx/sbl/blob/master/Chapter%203/fig_3.3.R), but the -2ll isn't even close. Lavaan matches Mplus perfectly. I wonder what's going on?

**Comment by mcneale** [comment]
Can we specify the model in wide format and see which -2 ln likelihood agrees?

**Comment by jpritikin** [comment]
> Can we specify the model in wide format and see which -2 ln likelihood agrees?

Yeah, that can help diagnose whether Rampart is working correctly or not.

> Also, do LRTs agree - maybe difference is a constant?

The parameter estimates are close, but I don't think they are close enough for this to be a plausible explanation. Doesn't Mplus/lavaan use a sufficient statistic approach? It's hard to believe that these other software are doing a raw data analysis in about 2-3 seconds.

**Comment by mcneale** [comment]
Ah perhaps they are taking a shortcut which isn’t entirely kosher.
